backend/routes/readme.md

```diff
@@ -0,0 +1,9 @@
+Full readme and front end tests can be found here - still under construction:
+
+https://github.com/spicycheeto/XRPL-NFT-Proof-of-Attendance-Infrastructure
+
+notes: 
+
+-memos are not attaching but single minting is coming back successful. 

```

Comment:
are those still the case?

Reply:
Yes, I believe this is an issue with the xumm walllet not displaying it. The memo is attached properly but doesnt come up in searches.